desktop/window-handlers/get-logs/index.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const { app, dialog } = require( 'electron' );
+
+/**
+ * Internal dependencies
+ */
+const state = require( 'lib/state' );
+const system = require( 'lib/system' );
+const { zipContents } = require( 'lib/archiver' );
+const log = require( 'lib/logger' )( 'desktop:get-logs' );
+
+/**
+ * Module variables
+ */
+const logPath = state.getLogPath();
+
+function pad( n ) {
+	return n < 10 ? '0' + n : n;
+}
+
+const localDateTime = () => {
+	const now = new Date();
+	return now.getFullYear() +
+		'-' +
+		pad( now.getMonth() + 1 ) +
+		'-' +
+		pad( now.getDate() ) +
+		'T' +
+		pad( now.getHours() ) +
+		'.' +
+		pad( now.getMinutes() ) +
+		'.' +
+		pad( now.getSeconds() ) +
+		'.' +
+		pad( now.getMilliseconds() );
+}
+
+module.exports = async function( window ) {
+	const onZipped = async ( file ) => {
+		await dialog.showMessageBox( window, {
+			type: 'info',
+			buttons: [ 'OK' ],
+			title: 'Logs saved to your desktop',
+			message: 'Logs saved to your desktop' +
+				'\n\n' +
+				`${ path.basename( file ) }`,
+			detail: 'For help with an issue, please contact help@wordpress.com and share your logs.'
+		} )
+	}
```

Comment:
fun fact: the curlies here aren't necessary. they are fine, but they are superfluous

```js
const onZipped = async ( file ) => await dialog.showMessageBox( window, {
  …
} );
```

Reply:
also, if `dialog.showMessageBox` returns a `Promise` and this function isn't doing anything with it, it might be superfluous to also wrap this in `async` - same with `onError` below.

won't the runtime be identical to if we declared these as non-async?

```js
const onZipped = file => dialog.showMessageBox( … );
```

Reply:
Yep - the async/await here is unnecessary, updated in bee3aad. Also fixed the callback as it was misspecified (should return a function).

Reply:
You don't need the curlies on L56 and L68, Arrow functions allow for the following syntax:

`const onError = ( error ) => dialog.showMessageBox( window, {...});`

Although that means that calling onError returns the result of dialog.showMessageBox rather than undefined.

Reply:
Remove unnecessary curlies in 5ae2f9bb927a1267b487945ecd96b0dcc639a135